src/src/Helpers/Networking.h

```diff
@@ -7,12 +7,14 @@
 #include <WiFiClient.h>
 #include <WiFiUdp.h>
 
+#if FEATURE_DOWNLOAD
```

Comment:
Nope, 
`#if FEATURE_HTTP_CLIENT`

Reply:
First copied it from the .cpp source, fixed now.